build.gradle.kts

```diff
@@ -55,8 +54,8 @@ kotlin {
         }
         val jvmTest by getting {
             dependencies {
-                implementation("org.jetbrains.kotlin:kotlin-test:$kotlinVersion")
-                implementation("org.jetbrains.kotlin:kotlin-test-junit:$kotlinVersion")
+                implementation("org.jetbrains.kotlin:kotlin-test")
```

Comment:
I learned from gradle folks at Kotlinconf that we don't need to specify kotlin version for these. It'll just the right version for the version of Kotlin you picked.